components/Blueprints/class-runner.php

```diff
@@ -383,7 +383,7 @@ private function load_blueprint() {
 		if ( function_exists( 'mb_check_encoding' ) ) {
 			$is_valid_utf8 = mb_check_encoding( $blueprint_string, 'UTF-8' );
 		} else {
-			$is_valid_utf8 = utf8_is_valid_byte_stream( $blueprint_string );
+			$is_valid_utf8 = ! _wp_has_noncharacters_fallback( $blueprint_string );
```

Comment:
note too, @adamziel that this is a semantic change. you might have wanted that, but `wp_has_noncharacters()` specifically indicates if a string contains Unicode noncharacters, which are mostly (but not exclusively) the last two code points in each plane (U+xxFFFE, U+xxFFFF). it may return `true` if there is invalid UTF-8.

`wp_is_valid_utf8()` indicates if the byte stream is valid UTF-8. it may return `true` if there are noncharacters.